packages/eslint-plugin/src/rules/no-unsafe-argument.ts

```diff
@@ -142,7 +142,7 @@ export default util.createRule<[], MessageIds>({
       unsafeArgument:
         'Unsafe argument of type `{{sender}}` assigned to a parameter of type `{{receiver}}`.',
       unsafeTupleSpread:
-        'Unsafe spread of a tuple type. The {{index}} element is of type `{{sender}}` and is assigned to a parameter of type `{{reciever}}`.',
+        'Unsafe spread of a tuple type. The argument is of type `{{sender}}` and is assigned to a parameter of type `{{receiver}}`.',
```

Comment:
This is an actual bug. The `{{index}}` placeholder was never provided for this message. It also doesn't seem necessary to me as the violation is already reported on the specific argument that it refers to.